db/migrate/20181227102117_create_users.rb

```diff
@@ -1,8 +0,0 @@
-class CreateUsers < ActiveRecord::Migration
```

Comment:
you can not delete previous migration if they was on master. You can only create new migrations, deleting previous is strickly prohibited.

Reply:
my fail

Reply:
Pls, return migration)